daemon/system-info.sh

```diff
@@ -30,6 +30,9 @@ if [ -z "${VIRTUALIZATION}" ]; then
                     VIRTUALIZATION="none"
             fi
     fi
+    if [ -z "${VIRTUALIZATION}" ]; then
+      VIRTUALIZATION="unknown"
```

Comment:
I think that `none` is the expected value instead of `unknown`. It affects the display on the Cloud view - @manos-saratsis will know.

Reply:
How does it change from the value on L14 to blank?

Reply:
I just copied it from line 14 of the same file. Also, `none` means something different than `unknown`, so I don't know what we should do about that.

Reply:
One of the commands could have errored, in the user's case it must have been `systemd-detect-virt -v`.

Reply:
It goes over the ACLK and changes the labels drawn on the node-view table. Give @manos-saratsis a chance to look into it

Reply:
L30 above sets it to none